influxdb-watcher/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.7 as influxdb-watcher-builder
+
+# To force a rebuild, pass --build-arg REBUILD="$(DATE)" when running
+# `docker build`
+ARG REBUILD=1
+
+ARG WATCHER_REPO=https://github.com/monasca/monasca-watchers
+ARG WATCHER_BRANCH=master
+
+ENV GOPATH=/go
+ENV PROJECT_DIR=$GOPATH/src/github.com/monasca/monasca-watchers
+
+RUN apk add --no-cache openssl-dev git go glide g++
+
+WORKDIR $PROJECT_DIR
+RUN git init && \
+    git remote add origin $WATCHER_REPO && \
+    git fetch origin $WATCHER_BRANCH && \
+    git reset --hard FETCH_HEAD
+
+RUN glide install && \
+    go build -o influxdb-watcher -tags static main/influxdb_watcher.go
+
+FROM alpine:3.7
```

Comment:
might be lack of knowledge but why the FROM alpine twice?

Reply:
Back from training.
Two steps dockerfiles create smaller end images because it contain only output binary without dependencies needed for building (so it's useful for tools that use static compilation - like written in Go - but not so useful for scripting languages - like Python)